docs/jam/jam/introduction.rst

```diff
@@ -10,6 +10,7 @@ Jam is a language frontend that translates to
 :ref:`Lekvar <lekvar-introduction>`, which can then be compiled or interpreted
 by the backend.
 
+Each section will feature a worded description, a formal syntax definition and where possible, coded examples to demonstrate that feature in Jam syntax.
```

Comment:
Currently all documentation wraps at 80. This makes it much easier to read the source.